src/routes/__tests__/shelter.routes.test.js

```diff
@@ -1,21 +1,54 @@
-import mongoose from 'mongoose';
 import request from 'supertest';
 import app from '../../app';
-import { connectToDatabase, closeDatabase } from '../../utils/database';
+import { connectToDatabase, clearDatabase, closeDatabase } from '../../utils/database';
+import Shelter from '../../models/shelter.model';
+
+const sheltersData = [
+  {
+    place_name: 'Test name',
+    address: 'Berlin, Turmstrasse, 0',
+    contact_person: 'Test',
+    phone: '+497777777777',
+    longitude: 12.12,
+    latitude: 13.13,
+    description: 'Test description',
+  },
+  {
+    place_name: 'Test name2',
+    address: 'Berlin, Wicherstr, 10',
+    contact_person: 'Test contact',
+    phone: '+497777779999',
+    longitude: 12.12,
+    latitude: 13.13,
+    description: 'Test description',
+  },
+];
+
+beforeEach(async () => {
+  const shelterTest1 = new Shelter(sheltersData[0]);
+  const shelterTest2 = new Shelter(sheltersData[1]);
+  await shelterTest1.save();
+  await shelterTest2.save();
```

Comment:
Why dont u use create here? Then we can make it shorter since Model.create is equivalent to new Shelter and then save()

Reply:
i had some error with create, but do not remember what was that, i will check again